app/assets/javascripts/staff/events.js

```diff
@@ -7,5 +7,9 @@ $(document).ready(function () {
   $('.cancel-status-change').click(function() {
     $('.status-dropdown').hide();
     $('.btn-nav').show();
-  })
-});
\ No newline at end of file
+  });
+
+  $('.modal').on('keypress', function (e) {
+    if (e.which == 13) $(this).submit();
+  });
```

Comment:
Modals are custom widgets; there's no universal default defined in any standard. Whatever library we use will probably have its own default behavior. Depending on the library, it may even have support for submitting forms contained in the modal.